### PR TITLE
update cosmjs Your First CosmJS Actions - Send Tokens

### PR DESCRIPTION
**Pull request by tacolopo**
Documentation content update for [Academy, CosmJS, Your First CosmJS Actions - Send Tokens / Compose Complex Transactions]

For Your First CosmJS Actions - Send Tokens, this PR fixes grammar on line 81, alters the rpc to prevent ssl fails I was running into, explicitly states where to change an address to your own, and provides notes to update node if key creation fails with npx. 

For Compose Complex Transactions, remediation for potential Gas Price and Address Not Found errors were added. 

The changes in this Pull Request include (please tick all that apply):

* [x] Small language/grammar fixes
* [x] Small content fixes 
* [x] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [x] completed

### R2: External review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [ ] Ready to be merged

### CODE REVIEW

File: academy/5-cosmjs/first-steps.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ generateKey()
 
 Now create a **key** for our imaginary user **Alice**:
 
+*Note: You likely need to update node to a later version if this fails. Guide here: https://phoenixnap.com/kb/update-node-js-version
```
Comment (xavierlepretre) [inline academy/5-cosmjs/first-steps.md:108]:
```suggestion
*Note: You likely need to update node to a later version if this fails. Find a guide [here](https://phoenixnap.com/kb/update-node-js-version).
```

File: academy/5-cosmjs/first-steps.md [hunk 2]
```diff
@@ -181,7 +183,7 @@ Normally you would not yet have access to your user's address. However, for this
 ```typescript [https://github.com/b9lab/cosmjs-sandbox/blob/723d2a9/experiment.ts#L10-L13]
 console.log(
     "Alice balances:",
-    await client.getAllBalances("cosmos17tvd4hcszq7lcxuwzrqkepuau9fye3dal606zf"),
+    await client.getAllBalances("cosmos17tvd4hcszq7lcxuwzrqkepuau9fye3dal606zf <replace with your generated address>"),
```
Comment (xavierlepretre) [inline academy/5-cosmjs/first-steps.md:186]:
```suggestion
    await client.getAllBalances("cosmos17tvd4hcszq7lcxuwzrqkepuau9fye3dal606zf"), // <-- replace with your generated address
```

File: academy/5-cosmjs/multi-msg.md [hunk 3]
```diff
@@ -259,7 +259,14 @@ export interface MsgDelegate {
 }
 ```
 
-Now that you know the `typeUrl` for delegating some tokens is `/cosmos.staking.v1beta1.MsgDelegate`, you need to find a validator's address that Alice can delegate to. Find a list of validators in the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/validators). Select a validator and use their address in the following script, which you can copy to replace your original token transfer:
+Now that you know the `typeUrl` for delegating some tokens is `/cosmos.staking.v1beta1.MsgDelegate`, you need to find a validator's address that Alice can delegate to. Find a list of validators in the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/validators). Select a validator and set their address as a variable:
+
+```
+const validator: string = "cosmosvaloper178h4s6at5v9cd8m9n7ew3hg7k9eh0s6wptxpcn" //01node
+
```
Comment (xavierlepretre) [inline academy/5-cosmjs/multi-msg.md:266]:
```suggestion
```

File: academy/5-cosmjs/multi-msg.md [hunk 4]
```diff
@@ -259,7 +259,14 @@ export interface MsgDelegate {
 }
 ```
 
-Now that you know the `typeUrl` for delegating some tokens is `/cosmos.staking.v1beta1.MsgDelegate`, you need to find a validator's address that Alice can delegate to. Find a list of validators in the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/validators). Select a validator and use their address in the following script, which you can copy to replace your original token transfer:
+Now that you know the `typeUrl` for delegating some tokens is `/cosmos.staking.v1beta1.MsgDelegate`, you need to find a validator's address that Alice can delegate to. Find a list of validators in the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/validators). Select a validator and set their address as a variable:
+
+```
```
Comment (xavierlepretre) [inline academy/5-cosmjs/multi-msg.md:264]: Can you add `typescript` after the 3 back ticks? So we get the correct code highlighting.

File: academy/5-cosmjs/multi-msg.md [hunk 5]
```diff
@@ -279,14 +286,45 @@ const result = await signingClient.signAndBroadcast(
...
                 amount: { denom: "uatom", amount: "1000", },
             },
           },
     ],
     "auto",
 )
 ```
+Note: If you get the error "Error: Gas price must be set in the client options when auto gas is used." You can fix it by changing the command to the following:
+
+```
```
Comment (xavierlepretre) [inline academy/5-cosmjs/multi-msg.md:299]: Here too `typescript`.

File: academy/5-cosmjs/multi-msg.md [hunk 6]
```diff
@@ -279,14 +286,45 @@ const result = await signingClient.signAndBroadcast(
...
-                validatorAddress: your_selected_validator_address,
+                validatorAddress: validator,
                 amount: { denom: "uatom", amount: "1000", },
             },
           },
     ],
     "auto",
 )
 ```
+Note: If you get the error "Error: Gas price must be set in the client options when auto gas is used." You can fix it by changing the command to the following:
```
Comment (xavierlepretre) [inline academy/5-cosmjs/multi-msg.md:297]: I would say to change the `"auto"` part to something else and give an example. Here it sounds too much like we have to change everything (scary).

**Review by xavierlepretre — CHANGES_REQUESTED** [review]

File: academy/5-cosmjs/multi-msg.md (hunk 4 above)
Comment (tacolopo) [reply academy/5-cosmjs/multi-msg.md]: added

File: academy/5-cosmjs/multi-msg.md (hunk 5 above)
Comment (tacolopo) [reply academy/5-cosmjs/multi-msg.md]: added

File: academy/5-cosmjs/multi-msg.md (hunk 6 above)
Comment (tacolopo) [reply academy/5-cosmjs/multi-msg.md]: removed command and changed auto

**Review by tacolopo — COMMENTED** [review]
All changes made.

File: academy/5-cosmjs/first-steps.md [hunk 7]
```diff
@@ -105,6 +105,8 @@ generateKey()
 
 Now create a **key** for our imaginary user **Alice**:
 
+*Note: You likely need to update node to a later version if this fails. Find a guide [here](https://phoenixnap.com/kb/update-node-js-version).
```
Comment (xavierlepretre) [inline academy/5-cosmjs/first-steps.md:108]:
```suggestion
*Note: You likely need to update Node.js to a later version if this fails. Find a guide [here](https://phoenixnap.com/kb/update-node-js-version).
```

**Review by xavierlepretre — COMMENTED** [review]
@AndrewB9lab Style review, Ser?

**Comment by tacolopo** [comment]
Made recommended corrections.

**Review by AndrewB9lab — APPROVED** [review]
Language review of changes performed, no edits necessary.

**Comment by coldice** [comment]
Thank you for your contribution @tacolopo, looks all good.

**Pull request merged**